src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,47 @@
 
 class Animal {
   // write your code here
+  static alive = [];
+
+  static removeDead() {
+    const aliveAnimals = this.alive.filter((animal) => animal.health > 0);
+
+    this.alive = [...aliveAnimals];
+  }
+
+  constructor(name, health) {
+    this.name = name;
+    this.health = health || 100;
```

Comment:
The parameters for the constructor are in the wrong order. According to the task description, the name should come first, followed by health. This needs to be fixed to match the expected API.